frontend/apps/web/src/pages/all-tests/index.tsx

```diff
@@ -0,0 +1,285 @@
+import { PageContainer } from '@/containers/PageContainer'
+import {
+  Button,
+  Colors,
+  Link,
+  PageLayout,
+  ProgressBar,
+  Table,
+  Text,
+  usePageLayout,
+} from '@filecoin/ui'
+import { styled } from '@storybook/theming'
+import React from 'react'
+import { useNavigate } from 'react-router-dom'
+
+const Dashboard = () => {
+  const {
+    state: { model },
+  } = PageContainer.useContainer()
+  const allTests = model.getAllTests()
+  const navigate = useNavigate()
+
+  const Header = props => {
+    const navigate = useNavigate()
+    return (
+      <PageLayout.Header>
+        <HeaderWrapper>
+          <Text type="heading 5">All Tests</Text>
+          <Buttons>
+            <Button
+              onClick={() => navigate('/repository-details')}
+              variant="outline"
+              size="medium"
+            >
+              All Behaviours
+            </Button>
+          </Buttons>
+        </HeaderWrapper>
+      </PageLayout.Header>
+    )
+  }
+
+  const prepareAllTestsChart = () => {
+    return Object.entries(
+      allTests.reduce(
+        (d, test) => {
+          if (test.kind === 'e2e') {
```

Comment:
This has a limitation, because if another kind is added it will not show here. So we need to handle that, for example `d[test.kind] = d[test.kind] || 0 + 1`

Reply:
```
Object.entries(
      allTests.reduce(
        (d, test) => {
          d[test.kind] = (d[test.kind] || 0) + 1
          return d
        },
        {},
      ),
    ).map(([key, count]) => ({
      name: key,
      percentage: (count / allTests.length) * 100,
    }))
```

Reply:
Done